app/views/shortFormatPreSentenceReport/page8.scala.html

```diff
@@ -43,6 +43,7 @@ <h1 class="heading-xlarge">Risk assessment</h1>
             @inputHidden(reportForm("onBehalfOfUser"), encrypter)
             @inputHidden(reportForm("entityId"), encrypter)
             @inputHidden(reportForm("documentId"), encrypter)
+            @inputHidden(reportForm("startDate"), encrypter)
```

Comment:
Move to page 11 (repeat all pages)

Reply:
Done. We also missd adding

        @inputHidden(reportForm("courtOfficePhoneNumber"))
        @inputHidden(reportForm("counterSignature"))

to all pages